ASIM/dev/ASimTester/ASimTester.csv

```diff
@@ -534,17 +534,17 @@ EventOwner,string,Optional,ProcessEvent,,,
 EventOwner,string,Optional,RegistryEvent,,,
 EventOwner,string,Optional,UserManagement,,,
 EventOwner,string,Optional,WebSession,,,
-EventProduct,string,Mandatory,AuditEvent,Enumerated,Azure|Windows|Exchange 365|Dataminr Pulse|Vectra Cloud,
-EventProduct,string,Mandatory,Authentication,Enumerated,Service Cloud|Auth0|CloudTrail|AAD|Azure Defender for IoT|M365 Defender for Endpoint|Security Events|Okta|PostgreSQL|OpenSSH|su|sudo|Vectra Cloud|SentinelOne,
+EventProduct,string,Mandatory,AuditEvent,Enumerated,Azure|WAF|Windows|Exchange 365|Dataminr Pulse|ISE|Vectra XDR|Meraki,
```

Comment:
Don't consider Event Vendor again into Event Product category, it can be like WAF, XDE, EDR, Firewall etc

Reply:
@anki-narravula thanks. I removed 'Vectra' from 'Vectra XDR'. Any other modification needed?
Also, since 'Vectra XDR' was used in parser, I would need to update the same in parser as well i.e. changing EventProduct from 'Vectra XDR' to 'XDR'